src/acceleration/AitkenAcceleration.hpp

```diff
@@ -36,6 +36,11 @@ class AitkenAcceleration : public Acceleration {
   virtual void iterationsConverged(
       const DataMap &cpldata);
 
+protected:
+  /// @brief Concatenates the data and old data in cplData into two long vectors
+  void concatenateCouplingData(
+      const DataMap &cplData, const std::vector<DataID> &dataIDs, Eigen::VectorXd &targetValues, Eigen::VectorXd &targetOldValues) const;
```

Comment:
```suggestion
  void concatenateCouplingData(
      const DataMap &cplData, const std::vector<DataID> &dataIDs, Eigen::VectorXd &targetValues, Eigen::VectorXd &targetOldValues) const override final;
```
Use `override final`. Compare to ([`hasAnySendData`](https://github.com/precice/precice/blob/15cdcbdf72fea5c06c996cc1c2e0f0b4e137916d/src/cplscheme/BiCouplingScheme.hpp#L65))